avocado_vt/plugins/vt_joblock.py

```diff
@@ -11,12 +11,20 @@
 from avocado.utils.process import pid_exists
 from avocado.utils.stacktrace import log_exc_info
 
-# Avocado's plugin interface module has changed location. Let's keep
-# compatibility with old for at, least, a new LTS release
 try:
-    from avocado.core.plugin_interfaces import JobPre, JobPost
+    try:
+        # JobPre has become JobPreTests on newer avocado versions
+        from avocado.core.plugin_interfaces import JobPreTests as Pre
+        # JobPost has become JobPostTests on newer avocado versions
+        from avocado.core.plugin_interfaces import JobPostTests as Post
```

Comment:
I think it'd make sense to only use the latest names (`JobPreTests`) and rename only the compatibility imports (`JobPre` as `JobPreTests`) because the old `JobPre` in fact works as `JobPreTests` and I think it'd be easier to follow the changes once the old API is not supported.